bmfuncts/coupling_analysis.py

```diff
@@ -276,35 +326,22 @@ def coupling_analysis(institute, org_tup, wf_path,
         The dict keyed by countries and valued by the list of normalyzed affiliation types \
```

Comment:
```suggestion
        The dict keyed by countries and valued by the list of normalised affiliation types \
```
-> same mistake l.145

Reply:
Taken into account in next commit. Thanks